docs/service-patterns/appointment.md

```diff
@@ -0,0 +1,260 @@
+- Submit evidence
+- Get help (lower confidence)
+
+#### 5. After appointment
+
+- Get appointment outcome (lower confidence)
+- Know how to complain or challenge the outcome (lower confidence)
+- Get support after the appointment (lower confidence)
+
+### Organisations managing appointments: sub-patterns and steps
+
+#### 1. Book appointment
+
+- Understand the person's details
+- Understand the appointment details (lower confidence)
+- Contact the person
+- Arrange the appointment
+- Allocate appointment to an owner (lower confidence)
+- Arrange reasonable adjustments (lower confidence)
+- Arrange translator or chaperone (lower confidence)
+- Notify the person of appointment details and how to prepare
+- Change or cancel appointment
+
+#### 2. Prepare for appointment
+
+- Communicate availability for appointments
+- View upcoming appointments
+- Understand the person's details
+- Understand details of a claim or application (lower confidence)
+- Arrange safeguarding (lower confidence)
+- Check the person has confirmed the appointment (lower confidence)
+- Review technical requirements of appointment (lower confidence)
+- View their information and history (lower confidence)
+- Send reminders (lower confidence)
+
+#### 3. Attend appointment
+
+- Verify person's identity (lower confidence)
+- Register them for appointment (lower confidence)
+- Brief the third party (lower confidence)
+- Carry out appointment
+- Record attendance (lower confidence)
+- Defer appointment (lower confidence)
+
+#### 4. After the appointment
+
+- Write up outcome (lower confidence)
+- Send outcome and any next steps (lower confidence)
+- Give support (lower confidence)
+
+{% endtab %}
+{% tab "User scenarios" %}
+
+## User scenarios
+
+These are some scenarios you may have to design for your service.
+
+### Happy paths:
+
+- Person books the appointment themselves
+- Someone else books it for them, for example a service provider
+- The person is invited to book it by a service provider
+- People get appointment reminders
+- Someone else books it for them, for example a service provider
+- The person is invited to book it by a service provider
+- User requests and receives assistance or adjustments to book or attend an appointment
+- The appointment happens at the right place, on time and runs to time
+- User or service provider can view information about the outcome of the appointment
+
+### Alternatives:
+
+
+### Unhappy paths:
+
+●	There are no suitable appointments  
+●	The user cannot book the appointment themselves
+
+The appointment cannot go ahead (or be completed) because:
+
+●	The user is late, does not attend or cancels at short notice
+●	The provider is late, does not attend or cancels at short notice
+●	There are technical issues or another problem during the appointment
+●	There are scheduling issues with multiple attendees  
+●	The user does not meet essential requirements, for example bringing evidence
+●	The appointment is not paid for
```

Comment:
```suggestion
 - There are no suitable appointments  
 - The user cannot book the appointment themselves

The appointment cannot go ahead (or be completed) because:

 - The user is late, does not attend or cancels at short notice
 - The provider is late, does not attend or cancels at short notice
 - There are technical issues or another problem during the appointment
 - There are scheduling issues with multiple attendees  
 - The user does not meet essential requirements, for example bringing evidence
 - The appointment is not paid for
```